Project-Descriptions-and-Plans/CV32E4_TBD/Project_Concept_for_CV32E4_TBD_Nov-27_2023.md

```diff
@@ -0,0 +1,250 @@
+# OpenHW Project Concept Proposal: CV32E4_TBD
+*The PC proposal introduces the project and explains the market drivers and the "why"*
+
+# Title of Project - "CV32E4_TBD"
+# Project Concept Proposal
+## Date of proposal - 2023-11-27
+## Author(s) - Manuel PEZZIN (CEA)
+
+## High Level Summary of project, project components, and deliverables
+This proposal is a continuation of CV32E40Pv2 project.
+Main goal is to support official RVB (and a part of RVK) and (upcoming) RVP
+extensions while keeping backward compatibility with CV32E40Pv1/v2 (eg XPULP
+custom extensions are kept and even extended).
+Additionnal features may also be considered, like:
+* CV-X-IF support
+* CLIC support
+* Improved OBI interface support (bus error in particular)
+
+This new project will keep functionnal backward compatibility with CV32E40Pv2,
+but with relaxed rules compared to previous project. In particular, formal equivalence
+with previous design will not be required to make design work easier.
+
+Due to the strict backward compatibility rules of CV32E40Pv2, this new core is not
+named CV32E40Pv3 and a new naming should be found.
+
+## Summary of market or input requirements
+
+
+### Known market/project requirements at PC gate
+
```

Comment:
The list of requirements is good. However, we are missing an explanation of where these requirements arise from. Can you please give an overall summary of the objective of the new core, that is what segment will it fill that differs from E40Pv2, E40X, etc.?

Reply:
The objective is to have a core tailored for embedded real-time control/DSP workloads.
My initial intent was was to propose a work plan for a new revision of one of these 2 cores that would support RVB and RVP extensions.
After an in-depth review, the outcome was that I would need to mix features from both designs.
Using CV32E40Pv2 as stating point would make this "merge" easier because the SIMD datapath is verified and will be a good starting point for adding RVP support.
Backporting RVB, RVK, CLIC and CV-X-IF from E40X was judged easier than re-introducing XPULP features to E40X

Reply:
Is it sufficient if I move the explication text that is at the end back to the first section ?

Reply:
@MPEZZIN that is fine